src/components/nanopub/ExportNanopub.tsx

```
@@ -0,0 +1,357 @@
+                      </td>
+                      <td style={{ verticalAlign: "middle" }}>
+                        <Button
+                          icon="eye-open"
+                          disabled={!nanopubSettings?.enabled}
+                          onClick={() => setPreviewNanopub(r)}
+                        />
+                      </td>
+                      <td style={{ verticalAlign: "middle" }}>
+                        <Tooltip
+                          content={
+                            !!nanopubProps?.published
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)